lib/pact/doc/interaction_view_model.rb

```diff
@@ -21,7 +21,7 @@ def id
             "#{description} given #{interaction.provider_state}"
           else
             description
-          end
+                      end
           CGI.escapeHTML(full_desc.gsub(/\s+/,'_'))
```

Comment:
Of all the changes, this is the one I find the most weird! I don't understand why the end is so far indented.

Reply:
This one is rubocop auto fixes. I'll find and correct them.
I thinks there are 2-3 places like this

Reply:
Was telling the lie :D. There're more than 3 places

Fixed e35fab94